tests/src/Database/DBStructureTest.php

```diff
@@ -45,7 +45,7 @@ protected function setUp(): void
 	 */
 	public function testExists() {
 		self::assertTrue(DBStructure::existsTable('user'));
-		self::assertFalse(DBStructure::existsTable('notatable'));
+		self::assertFalse(DBStructure::existsTable('notable'));
```

Comment:
This would also work, but in the original test, this should have been `not_a_table`. `missingtable` or `missing_table` might be a better choice.

Reply:
I tend to use `nonexistent`